### DISCUSSION TITLE
Why is it `expession` (string) instead of a function?

### DISCUSSION THREAD

**Discussion by saneef** [neocarto/bertin#81] [Q&A]
In [GeoJSON properties selection](https://github.com/neocarto/bertin#geojson-properties-selections), a parameter is `expression`. I'm curious why is an `expression` (a string) preferred over a pure JavaScript function?

**Comment by neocarto** [comment]
All the parameters of the package are strings (except for data) because I make sure that this library can be easily transposed in the ecosystem of R, which is not like I JS, a purely functional language. See https://github.com/riatelab/bertin. But you are right. We have to accept functions too. That's what I started to do in the tooltips. See section 4 https://observablehq.com/@neocartocnrs/bertin-js-tooltips?collection=@neocartocnrs/bertin
